docs/Style.thy

```diff
@@ -359,6 +359,66 @@ term "
   A" \<comment> \<open>NOT OK: implies this parses as @{text "((A \<and> B) \<longrightarrow> B) \<or> A"}\<close>
 
 
+section \<open>Wrapping/indenting complex rule instantiations, usually as part of @{text "_tac"} methods\<close>
+
+text \<open>
+  For simple instantiations, they can fit it on one line, like so.
```

Comment:
This sentence is a bit unclear to me. Are you saying that simple instantiations that can fit on one line should look like the following?

Reply:
Yes. I should make that more explicit.